src/lib/components/forms/AddOnDispatch.svelte

```diff
@@ -51,6 +52,12 @@
   $: validator = ajv.compile({ type: "object", properties, required });
   $: hasEvents = eventOptions && eventOptions.events.length > 0;
   $: hasFields = Object.keys(properties).length > 0;
+  // prefill values from search params
+  $: new URLSearchParams($page.url.searchParams).forEach((v, k) => {
```

Comment:
I think we want to do this in `afterNavigate` so it doesn't run any time the URL changes and overwrite what the user has filled in.